Paco-Server/src/com/google/sampling/experiential/server/CloudSqlSearchServlet.java

```diff
@@ -141,6 +145,12 @@ public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOEx
           return;
         }
         
+        if (sqlQueryObj.getCriteriaQuery() == null) {
+          sendErrorMessage(resp, mapper,
+                           ErrorMessages.EXPERIMENT_ID_CLAUSE_EXCEPTION.getDescription() + postBodyString);
```

Comment:
Is this the correct error message to report back? It seems like we are trying to say that they did not provide any search criteria at all and therefore we cannot execute the query. If that is correct, let's create a new ErrorMessage just for that.

Reply:
It makes sense to have a new message. You are correct.